### ISSUE TITLE
 Message type "caffe.LayerParameter" has no field named "batch_norm_param"

### ISSUE THREAD

**Issue by wolfbill** [BVLC/caffe#4136]
Hi man:
     When I run ./train_full_sigmoid_bn.sh. I got the error list below:
    I0512 15:24:13.964053 23252 layer_factory.hpp:74] Creating layer bn1
    F0512 15:24:13.964093 23252 layer_factory.hpp:77] Check failed: registry.count(type) == 1 (0 vs.      1)   Unknown layer type: BatchNorm (known types: AbsVal, Accuracy, ArgMax, BNLL, Concat, ContrastiveLoss, Convolution, Data, Deconvolution, Dropout, DummyData, Eltwise, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, LRN, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Pooling, Power, Python, ReLU, Reduction, Reshape, SPP, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, WindowData)

I don't know where the problem is. Any one have any idea will be  appreciative

**Comment by wqlw** [comment]
l have the same question,Do you solute it please?

**Comment by wolfbill** [comment]
Yes,upgrade the new vision of caffe.

**Comment by wqlw** [comment]
l used the new one,but the question has been solved now.@wolfbill
